IssueTracker/Validation/CredentialsValidation.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Models.Response;
+
+
+namespace Validation
+{
+    public class CredentialsValidation
+    {
+        public static bool isValid(Credentials credentials)
+        {
+            if (credentials == null) return false;
+            if (string.IsNullOrEmpty(credentials.NameEmail) || string.IsNullOrEmpty(credentials.Password))
```

Comment:
what is NameEmail?

Reply:
the name or the email of the user. Since the user can log in with either the UserName or the Email

Reply:
then you should have 2 properties(username, email), and test if one of them is not null